### PR TITLE
[7.6.0][CDE] ApiGradientStop.ToJSON check

### PR DESCRIPTION
**Pull request by Yar04ek**

### CODE REVIEW

File: js/docx/smoke/api_gradient_stop/to_json.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+builder.CreateFile("docx");
+var oDocument = Api.GetDocument();
```
Comment (askonev) [inline js/docx/smoke/api_gradient_stop/to_json.js:2]: Codestyle plz.

**Review by askonev — CHANGES_REQUESTED** [review]

File: js/docx/smoke/api_gradient_stop/to_json.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+var oParagraph = oDocument.GetElement(0);
+var oGs1 = Api.CreateGradientStop(Api.CreateRGBColor(255, 111, 61), 0);
+var oGs2 = Api.CreateGradientStop(Api.CreateRGBColor(255, 213, 191), 100000);
+var oFill = Api.CreateRadialGradientFill([oGs1, oGs2]);
+var jsonFill = oFill.ToJSON();
```
Comment (askonev) [inline js/docx/smoke/api_gradient_stop/to_json.js:7]: Why do you write a test for the [ApiGradientStop](https://api.onlyoffice.com/docbuilder/textdocumentapi/apigradientstop/tojson) class object but convert the Fill object to JSON?

**Review by askonev — CHANGES_REQUESTED** [review]

File: spec/docx/smoke/api_gradient_stop_spec.rb [hunk 3]
```diff
@@ -6,4 +6,21 @@
     docx = builder.build_and_parse('js/docx/smoke/api_gradient_stop/get_class_type.js')
     expect(docx.elements.first.nonempty_runs[1].text).to eq('Class Type = gradientStop')
   end
+
+  it 'ApiGradientStop | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_gradient_stop/to_json.js')
+    json = JSON.parse(docx.elements[1].nonempty_runs.first.text)
+    grad_s = json['graphic']['spPr']['fill']['fill']['gsLst'][0]['type']
```
[the remark this answers is not in the document]
Comment (Yar04ek) [reply spec/docx/smoke/api_gradient_stop_spec.rb]: Fix in the new commit.

File: js/docx/smoke/api_gradient_stop/to_json.js (hunk 1 above)
Comment (Yar04ek) [reply js/docx/smoke/api_gradient_stop/to_json.js]: Fix in the new commit.

File: js/docx/smoke/api_gradient_stop/to_json.js (hunk 2 above)
Comment (Yar04ek) [reply js/docx/smoke/api_gradient_stop/to_json.js]: Fix in the new commit.

**Review by alinasafonova — APPROVED** [review]

**Review by askonev — APPROVED** [review]

**Pull request merged**